mailbox/api/src/main/java/org/apache/james/mailbox/MailboxManager.java

```diff
@@ -102,7 +102,8 @@
          *  provide an index on the fields: 
          *  From, To, Cc, Bcc, Subjects, textBody & htmlBody
          */
-        Text
+        Text,
+        Attachment
```

Comment:
We are mixing things here : what if I index Attachment but don't support fulltext on that index ?